src/DotnetRelease/OSPackagesRecords.cs

```diff
@@ -0,0 +1,85 @@
+using System.ComponentModel;
+using System.Text.Json.Serialization;
+
+namespace DotnetRelease;
+
+// For os-packages.json file
+// List of packages required by product
+// Example: https://github.com/dotnet/core/blob/main/release-notes/9.0/supported-os.json
+[Description("The set of packages required by a given product version for a set of distros.")]
+public record OSPackagesOverview(
+    [property: Description("Major (or major.minor) version of product.")]
+    string ChannelVersion,
+
+    [property: Description("Set of nominal packages used by product, with descriptions.")]
+    IList<Package> Packages, 
+    
+    [property: Description("Set of distributions where the product can be used.")]
+    IList<Distribution> Distributions);
+
+[Description("A nominal package is a distro-agnostic representation of a package, including the scenarios for which the package is required. A nominal package will be referenced by a distribution package, with a distribution-specific package name.")]
+public record Package(
+    [property: Description("ID of nominal package.")]
+    string Id,
+        
+    [property: Description("Display name of nominal package.")]
+    string Name,
+
+    [property: Description("Required scenarios for which the package must be used.")]
+    IList<Scenario> RequiredScenarios,
+
+    [property: Description("Minimum required version of library.")]
+    string? MinVersion = null,
+
+    [property: Description("Related references.")]
+    IList<string>? References = null);
+
+[Description("An operating system distribution, with required package install commands and specific packages for distribution releases.")]
+public record Distribution(
+    [Description("Name of the distribution, matching ID in /etc/os-release, however, the expectation is that this value starts with a capital letter (proper noun).")]
+    string Name,
+    
+    [Description("Commands required to install packages within the distribution.")]
+    IList<Command> InstallCommands,
+    
+    [Description("Releases for that distribution.")]    
+    IList<DistroRelease> Releases);
+
+[Description("A command to be run to install packages")]
+public record Command(
+    [Description("Whether the command needs to be run under sudo.")]    
+    bool RunUnderSudo,
+    
+    [Description("The command to be run, like apt.")]    
+    string CommandRoot,
+        
+    [Description("The command parts or arguments that need to be used.")]    
+    IList<string>? CommandParts = null);
+
+[Description("A distribution release with a list of packages to install.")]
+public record DistroRelease(
+    [Description("The name of the release, matching PRETTY_NAME in /etc/os-release.")]    
+    string Name,
+    
+    [Description("The version number for the release, matching VERSION_ID in /etc/os-release.")]    
+    string Release, IList<DistroPackage> Packages);
```

Comment:
You missed the documentation of `Packages` as it was on the same line.
```suggestion
    string Release,
    
    [Description("TODO")]    
    IList<DistroPackage> Packages);
```